jetty-client/src/main/java/org/eclipse/jetty/client/HttpClient.java

```diff
@@ -1080,14 +1112,44 @@ public void setStrictEventOrdering(boolean strictEventOrdering)
         this.strictEventOrdering = strictEventOrdering;
     }
 
+    /**
+     * @return the delay in ms before idle destinations should be removed
+     * @see #setDestinationIdleTimeout(long)
+     */
+    @ManagedAttribute("The delay in ms before idle destinations are removed, disabled when zero or negative")
+    public long getDestinationIdleTimeout()
+    {
+        return destinationIdleTimeout;
+    }
+
+    /**
+     * Whether destinations that have no connections (nor active nor idle) should be removed
```

Comment:
no connections and no exchanges queued.

Please use Javadoc standards of wrapping every paragraph with <p> and </p> (let's not use the old style of unclosed <p> as if it was a newline).

Reply:
Done.